chart/prometheus-msteams/templates/deployment.yaml

```diff
@@ -47,6 +47,9 @@ spec:
           args:
             - --config=/etc/config/connectors.yaml
             - --template-file=/etc/template/card.tmpl
+            - --idle-conn-timeout=90s
+            - --max-idle-conns=100
+            - --tls-handshake-timeout=30s
```

Comment:
Remove these values and everything should be good to be merged. There is no need to add it here.

@Knappek just ensure that we document the possibility to use these flags in the helm chart README.